src/components/Apps/AccountListTable/AccountList.vue

```diff
@@ -434,6 +437,7 @@ export default {
     },
     addAccountSuccess() {
       Reflect.deleteProperty(this.$route.query, 'flag')
+      this.isUpdateAccount = false
       this.$refs.ListTable.reloadTable()
     },
     async getAssetDetail() {
```

Comment:
As of October 2023, there are no known inconsistencies, potential issues or optimizations that need to be addressed with these codes provided.